misc/bench.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Hmm.. but I think keeping POSIX shell behavior would be better for a this kind of small script.  What about removing 'function' keyword, is that all it complains?

Reply:
I have tried it but there are more bash syntaxes so it still complains.

Reply:
Ah, ok.  Let's change it to `bash` then.